elpy.el

```diff
@@ -1259,7 +1262,12 @@ PyPI, or nil if that's VERSION."
 
 (defun elpy-insert--pip-button-action (widget &optional _event)
   "The :action option for the pip button widget."
-  (async-shell-command (widget-get widget :command)))
+  (with-pyvenv-deactivated
+   (sudo-async-shell-command (widget-get widget :command))))
+
+(defun sudo-async-shell-command (command)
+  (let ((default-directory "/sudo::"))
+    (async-shell-command command)))
```

Comment:
Why do we need sudo?
Also, seems it would only work with linux machines

Reply:
Admin rights would be needed to install global packages from Emacs (with the buttons in `elpy-config`).
Thanks for pointing this out, I didn't thought about other OS...
We may have to drop the `elpy-config` buttons in that case.